lib/screen/customer_support/support_list_page.dart

```diff
@@ -217,6 +242,65 @@ class _SupportListPageState extends State<SupportListPage>
     );
   }
 
+  Widget _announcementRow(Announcement announcement, bool hasDivider) {
+    final theme = Theme.of(context);
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Row(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            Expanded(
+              child: Row(
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  Text(
+                    announcement.getListTitle(),
+                    style: theme.textTheme.ppMori400Black16,
+                  ),
+                  if (announcement is AnnouncementLocal &&
+                      !announcement.read) ...[
+                    _unread(),
```

Comment:
should be `unreadWidget` or something else, I thought it was an action instead of widget

Reply:
fixed